### ISSUE TITLE
[Suggestion] group permission info aliases

### ISSUE THREAD

**Issue by voodootje0** [lucko/LuckPerms#473]
Dear,

Could you add `list` as aliases of /lp group <group> permission info? Maybe also for /lp user <user> permission info.

If you're going to change it, maybe it will break https://github.com/lucko/LuckPerms/issues/461 would be amazing if you can let both works.

**Comment by spannerman79** [comment]
> Could you add list as aliases of /lp group permission info? Maybe also for /lp user permission info.

Do it yourself https://bukkit.gamepedia.com/Commands.yml

**Comment by lucko** [comment]
I think that's a bit beyond the scope of what Bukkit can do @spannerman79 .

**Comment by spannerman79** [comment]
Actually it isn't @lucko 
```
aliases:
    list:
    - "lp user $$1 permission info"
    listg:
    - "lp group $$1 permission info"
```

`list` & `listg` can be subbed with anything you want or what isn't already taken.

**Comment by lucko** [comment]
You're misunderstanding the request (I think, at least :P)

The suggestion is to add `/lp group permission list` as an alias of `/lp group permission info`.

**Comment by spannerman79** [comment]
Either

> Could you add list as aliases of /lp group permission info? Maybe also for /lp user permission info.

Do what I already stated https://github.com/lucko/LuckPerms/issues/473#issuecomment-329919911

or

> If you're going to change it, maybe it will break #461 would be amazing if you can let both works.

What you interpreted.  OP has conflicting statements/requests.

----
That alias doesn't save any text/character length.  The only thing that it looks like is a very similar pex command.  If this project was suppose to take over from pex then fair enough - although in that case you wouldn't have spent the time to add in this wiki page https://github.com/lucko/LuckPerms/wiki/GM-&-PEX-Command-Equivalents

Therefore - conflicting statements/request by OP.

**Comment by voodootje0** [comment]
@spannerman79 You're misunderstanding me.

@lucko Exactly, an extra aliases (that doesn't conflict with #461)

**Comment by lucko** [comment]
We're able to revert the fix for #461 (just removing the alias) thanks to 1d1cd679dd516b51c4ad85208f708776a65a6101
